Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -421,7 +586,7 @@ public class HTTPClient {
     /// `HTTPClient` configuration.
     public struct Configuration {
         /// TLS configuration, defaults to `TLSConfiguration.forClient()`.
-        public var tlsConfiguration: TLSConfiguration?
+        public var tlsConfiguration: Optional<TLSConfiguration>
```

Comment:
Is it some auto formatting went wrong?

Reply:
@avolokhov No, sadly in classes/structs, `Optional<X>` is different to `X?`.

- `Optional<X>` needs to be manually initialised in `init` (or else compiler error)
- `X?` magically gets pre-initialised to `nil` first. So if you forget to set it in `init` you will _not_ get a compiler error :'( . Also it's slightly less efficient because if you do set it, it gets initialised twice. I know it doesn't make any sense but it is true: [SR-11777](https://bugs.swift.org/browse/SR-11777)

Reply:
😜

Reply:
Sadly, we've got a problem: https://github.com/nicklockwood/SwiftFormat/issues/636

SwiftFormat wants to force me to use the wrong one. I think I need to learn how to configure SwiftFormat to not do that.

Reply:
@nicklockwood responded super quickly, so I added `--disable typeSugar` which fixes this and leaves the `Optional<Foo>`s alone.